### PR TITLE
Cows

### PR DESCRIPTION
**Pull request by logatsang**
please

### CODE REVIEW

File: src/main/java/Cow.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        // Check if feeding is allowed
+        if (this.size == 10)
+            return false;
+
+        this.size += 1;
+        return true;
+    }
+
+    /**
+     * Feed the cow a fixed amount of feed, increasing its size.
```
Comment (edwardsues) [inline src/main/java/Cow.java]: should it be Starve the cow?

File: src/main/java/Cow.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
+/**
+ * The COW is a noble animal.
+ * Cows are nice. I like cows because they are a funny animal.
```
Comment (edwardsues) [inline src/main/java/Cow.java]: unnecessary comments

**Review by edwardsues — COMMENTED** [review]
See Reviews

**Review by edwardsues — CHANGES_REQUESTED** [review]
Check reviews concerning your documentation

File: src/main/java/Cow.java (hunk 1 above)
Comment (logatsang) [reply src/main/java/Cow.java]: i  am sorry i am bad at 
programing

File: src/main/java/Cow.java (hunk 2 above)
Comment (logatsang) [reply src/main/java/Cow.java]: hn

Comment (edwardsues) [reply src/main/java/Cow.java]: cry about it

**Comment by logatsang** [comment]
please it  is good now p

**Review by logatsang — COMMENTED** [review]
y

**Pull request merged**